allennlp_config/citation_classification.jsonnet

```diff
@@ -0,0 +1,56 @@
+{
+  "random_seed": std.parseInt(std.extVar("SEED")),
+  "pytorch_seed": std.parseInt(std.extVar("PYTORCH_SEED")),
+  "numpy_seed": std.parseInt(std.extVar("NUMPY_SEED")),
+  "dataset_reader": {
+    "type": "classification_dataset_reader",
+  },
+  "train_data_path": std.extVar("CITATION_TRAIN_PATH"),
```

Comment:
if you want exp.sh to be usable for all config files, then path here and in other config files will need to change. Maybe this is a different PR

Reply:
done